### DISCUSSION TITLE
Path has 2 positional parameters, but spec has 4 declared

### DISCUSSION THREAD

**Discussion by samirkape** [oapi-codegen/oapi-codegen#858] [Q&A]
Hi All,

While generating types from a spec file, I am getting below error:

`error creating operation definitions: path '/customers/{cid}/subscriptions/{subscriptionId}' has 2 positional parameters, but spec has 4 declared`

spec linter does not complain about any such issue and I also manually verified it to be as expected. 

Please help

**Comment by ghost** [comment]
Can you file an issue and attach the spec? It's hard to figure these out without the code.
